documentation/SERVICE_NOW/SERVICE_NOW.md

```diff
@@ -33,22 +35,110 @@ See the Prerequisites section below for instructions on how to obtain your Servi
 - URL: `https://dev12345.service-now.com`
 - Instance ID: `dev12345`
 
-### Create a ServiceNow Integration User
+### Setup ServiceNow Integration User
 
 1. Log in to your ServiceNow instance as an administrator
 2. Navigate to User Administration > Users
-3. Click "New" to create a new user
-4. Fill in the required fields:
-   - User ID: `aws_integration` (recommended)
-   - First Name: `AWS`
-   - Last Name: `Integration`
-   - Password: Create a secure password
-5. Assign the following roles:
-   - `admin` (or a custom role with permissions to create business rules)
-   - `incident_manager`
+3. If you do not have a user, click "New" to create a new user
```

Comment:
suggestion: Emphasize that creating an non-person entity (NPE) orphaned from a User enable the NPE to not be tied to a person that could leave the organization